python/python-core/classes-ii/classes-ii-discussion.md

```diff
@@ -0,0 +1,93 @@
+---
+author: Stefan-Stojanovic
+
+tags:
+  - discussion
+
+type: normal
+
+category: discussion
+
+---
+
+# Test Yourself
+
+---
+
+## Content
+
+Now that you've learned about classes, it's time to practice!
+
+How would you create and instantiate a class called `Rectangle`?
+
+The class should have an `__init__` method, plus two methods that will be used to calculate the area and perimeter of the rectangle.
+
+When you call the methods, they should print the calculated output along with a message.
+
+To achieve this, you should use the following concepts:
+- class keyword (`class`)
+- function creation (`def`..)
+- area of a rectangle (`Length * Width`)
+- perimeter of a rectangle (`2 * Length + 2 * Width`)
+
+If you're not sure what to do, check out this footnote[1].
+
+When you're ready, leave a comment here with your submission. If you don't want to participate, you can always check the comment section to see what the other users come up with.
+
+> 💡 Before you submit your program, take a look at [how you can format your comments](https://www.enki.com/glossary/general/markdown-formatting).
+
+> 💡 The guidelines above are just suggestions. Feel free to include other concepts in your solution as you see fit, the implementation is up to you.
+
+
+---
+
+## Footnotes
+
+[1: Hints]
+
+Start by creating the `Rectangle()` class and the `__init__` method:
+
+```python
+class Rectangle():
+  def __init__(self, l, w):
+      self.length = l
+      self.width = w
+```
+
+Then, add the two methods:
+
+```python
+# Example 1
+def perimeter(self):
+    return print("The perimeter of the given rectangle is",\
+                  self.length * 2 + self.width * 2)
+
+def area(self):
+    return print("The area of the given rectangle is",\
+                  self.length * self.width)
+
+
+# Example 2
+def perimeter(self):
+    return self.length * 2+ self.width * 2
+
+def area(self):
+    return self.length * self.width
```

Comment:
imo we can use two spaces for indentation. some of these lines are long and it will make following the code blocks a bit harder in the app

Reply:
Roger, this was copied after a repl test and I forgot to check after the upload